src/styles/styles.ts

```diff
@@ -211,8 +214,8 @@ const webViewStyles = (theme: ThemeDefault) =>
         },
     } satisfies WebViewStyle);
 
-const styles = (theme: ThemeDefault) =>
-    ({
+const styles = (theme: ThemeDefault) => {
+    const stylez: Styles = {
```

Comment:
This looks like a typo

Reply:
Let's use a different variable name that describes what this is

Reply:
Lol, this is not actually a typo. The reason I put this here is so that some styles can compose/reference other styles. It's basically like this:

```js
function styles() {
    const stylez = {
        blueBackground: {
            backgroundColor: 'blue',
        },
        whiteBorder: {
            border: '1px white',
        },
    };
    
    stylez.blueButton = [
        stylez.blueBackground,
        stylez.blueBorder,
    ];
    
    return stylez;
}
```

The only reason it's not just called `styles` is because that causes a lint error because `styles` is already defined as the name of the function.